pkg/controllers/status/work_status_controller.go

```diff
@@ -560,7 +561,7 @@ func (c *WorkStatusController) SetupWithManager(mgr controllerruntime.Manager) e
 func (c *WorkStatusController) eventf(object *unstructured.Unstructured, eventType, reason, messageFmt string, args ...interface{}) {
 	ref, err := util.GenEventRef(object)
 	if err != nil {
-		klog.Errorf("Ignore event(%s) as failing to build event reference for: kind=%s, %s due to %v", reason, object.GetKind(), klog.KObj(object), err)
+		klog.ErrorS(err, "Ignoring event. Faild to build event reference for: kind=%s, %s", "reason", reason, "kind", object.GetKind(), "reference", klog.KObj(object))
```

Comment:
```suggestion
		klog.ErrorS(err, "Ignoring event. Failed to build event reference.", "reason", reason, "kind", object.GetKind(), "reference", klog.KObj(object))
```
A typo and get rid of the `%s`.